woocommerce/woocommerce
# Product Reviews: not use useBlockProps twice

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





During the review of #46785, @ObliviousHarmony noticed that there is a similar issue with the `Product Reviews` block. Using  the `useBlockProps` function causes an unnecessary duplicate structure:

![5FtY13.png](https://github.com/woocommerce/woocommerce/assets/4463174/0c7ac2e0-d40c-4dab-adc6-5bcf15144567)


This PR removes the unnecessary duplicate structure:

![image](https://github.com/woocommerce/woocommerce/assets/4463174/c62caa3f-a717-495d-9a7a-67a08fb4cb46)





### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go on `/wp-admin/site-editor.php?postType=wp_template&postId=woocommerce%2Fwoocommerce%2F%2Fsingle-product`.
2. Add the Product Reviews block.
3. Open the Chrome Dev tools.
4. Click on `Elements`
5. Search `[data-type="woocommerce/product-reviews"]`
6. Ensure that you get only one result:

![image](https://github.com/woocommerce/woocommerce/assets/4463174/2cb32fbb-9fd3-47ef-a52c-a0ea4db5f4fc)



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

Product Reviews: not use useBlockProps twice

</details>
